cmake/Modules/FindSpdlog_EP.cmake

```diff
@@ -28,50 +28,60 @@
 # This module defines:
 #   - SPDLOG_INCLUDE_DIR, directory containing headers
 #   - SPDLOG_FOUND, whether Spdlog has been found
-#   - The Spdlog::Spdlog imported target
-
-# Search the path set during the superbuild for the EP.
-set(SPDLOG_PATHS ${TILEDB_EP_SOURCE_DIR}/ep_spdlog/include)
+#   - The spdlog::spdlog_header_only imported target
 
+# First try the CMake find module.
 if (NOT TILEDB_FORCE_ALL_DEPS OR TILEDB_SPDLOG_EP_BUILT)
-  find_path(SPDLOG_INCLUDE_DIR
-    NAMES spdlog/spdlog.h
-    PATHS ${SPDLOG_PATHS}
-    ${TILEDB_DEPS_NO_DEFAULT_PATH}
-  )
-endif()
+  if (TILEDB_SPDLOG_EP_BUILT)
+    # If we built it from the source always force no default path
+    SET(TILEDB_SPDLOG_NO_DEFAULT_PATH NO_DEFAULT_PATH)
+  else()
+    SET(TILEDB_SPDLOG_NO_DEFAULT_PATH ${TILEDB_DEPS_NO_DEFAULT_PATH})
+  endif()
 
-include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(Spdlog
-  REQUIRED_VARS SPDLOG_INCLUDE_DIR
-)
+  find_package(spdlog
+          PATHS ${TILEDB_EP_INSTALL_PREFIX}
+          ${TILEDB_SPDLOG_NO_DEFAULT_PATH}
+          )
+  set(SPDLOG_FOUND ${spdlog_FOUND})
+endif()
 
-if (NOT SPDLOG_FOUND AND TILEDB_SUPERBUILD)
-  message(STATUS "Adding Spdlog as an external project")
-  ExternalProject_Add(ep_spdlog
-    PREFIX "externals"
-    URL "https://github.com/gabime/spdlog/archive/v0.16.3.zip"
-    URL_HASH SHA1=00a732da1449c15b787491a924d63590c1649710
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ""
-    INSTALL_COMMAND ""
-    UPDATE_COMMAND ""
-    LOG_DOWNLOAD TRUE
-    LOG_OUTPUT_ON_FAILURE ${TILEDB_LOG_OUTPUT_ON_FAILURE}
-  )
-  list(APPEND TILEDB_EXTERNAL_PROJECTS ep_spdlog)
-  list(APPEND FORWARD_EP_CMAKE_ARGS
-    -DTILEDB_SPDLOG_EP_BUILT=TRUE
-  )
+if (NOT SPDLOG_FOUND)
+  if(TILEDB_SUPERBUILD)
+    message(STATUS "Adding spdlog as an external project")
+    ExternalProject_Add(ep_spdlog
+            PREFIX "externals"
```

Comment:
Looks like you need to set `INSTALL_DIR` here for the new version, at least to fix the R/mingw builds. It's failing because spdlog is installed in `Program Files`:

```
2020-12-28T12:57:28.2901660Z -- Installing: C:/Program Files (x86)/spdlog/include
2020-12-28T12:57:28.2915513Z -- Installing: C:/Program Files (x86)/spdlog/include/spdlog
...
```